src/trio/_path.py

```diff
@@ -199,6 +199,7 @@ def generate_iter(cls, attrs: dict[str, object]) -> None:
                 setattr(cls, attr_name, wrapper)
 
 
+# TODO: `pathmod` points to `os.path`... is that what we want?
```

Comment:
I left this because ATM `trio.Path` has a `pathmod` pointing to a synchronous path module. Which is weird but also we don't have an equivalent module IIRC.

Reply:
This might get changed, see https://github.com/python/cpython/pull/113893.

Reply:
https://github.com/python/cpython/pull/113893 was merged

Reply:
Looks like the TODO is irrelevant because AFAICT the `pathmod` or `flavour` or `flavor` or whatever it will be only does operations like joining paths. Seemingly nothing that interacts with FS?

Reply:
Yea, as far as I remember looking at it a bit ago I think they've just make an abstract class for the path interface, at least in that particular PR